### DISCUSSION TITLE
dbus-serialbattery development and debugging process

### DISCUSSION THREAD

**Discussion by Flydroid** [Louisvdw/dbus-serialbattery#901] [General]
Hi,

To start of: thank you for this awesome project! I'm working on adding support for a custom battery managment which I have developed. The code is well structured and adding the support for it doesn't seem to be to complicated.

One thing I do struggle with is developing on the raspberry pi running venus os. 
Only with a reboot the driver seems to use any changes to the files in /data/etc/dbus-serialbattery. I have tried restart-driver.sh but it doesn't seemt work. 

What is the best method to restart the driver for fast debugging?

**Comment by mr-manuel** [comment]
Just run `/data/etc/dbus-serialbattery/reinstall-local.sh`

**Reply by Flydroid** [reply]
Thank you @mr-manuel! 
That works!
